### DISCUSSION TITLE
ONVIF Client Test Tool "Less than three conformant devices from different manufacturers found"

### DISCUSSION THREAD

**Discussion by Je34** [onvif/testspecs#16] [Q&A] [answered]
Hello,
I want to certificate our CCTV software with Profile S and Profile G. I use the ONVIF Client Test Tool 21.06 rev. 4066.
About the devices used for the conformance: 
- AXIS Q6054-E firmware 8.40.4.2
- Bosch AUTODOME IP starlight 7000i firmware 7.61.0026
- HIKVISION DS-2DF6A836X-AEL firmware V5.6.0 build 190320
I found the Feature List XML files associated with the devices on the ONVIF website.
I ran the Conformance Test and all seems to be ok to be compliant for Profile S and G.
But when I want to submit the Conformance Files on the website (https://www.onvif.org/member-portal/add-update-products), I have an error message "Less than three conformant devices from different manufacturers found" and I don't understand the reason because I have 3 devices from 3 different manufacturers (one from AXIS, one from BOSCH and one from HIKVISION). 
Thanks for your help.

**Comment by Johan-Svensk** [comment]
Hi!
The problem is probably that the FW in **HIKVISION DS-2DF6A836X-AEL** is not conformance tested. 
You have used V5.6.0 build 190**320** and the latest approved FW for this model is V5.6.0 build 190**125**

So unfortunately you will have to get FW version V5.6.0 build 190125 from the device vendor and run the tests again.

Hope it helps!
Regards,
Johan

**Reply by Je34** [reply]
Thanks for the reply, 
I'll try if I can have the specific firmware (not available on the manufacturer Website) and I'll give you some news to confirm if the problem is about this point.
Regards,

**Comment by Je34** [comment, marked as answer]
Hi, I'm still having the problem. 
Is it possible to know which device is marked as non conformant? Because I only have the error line  "Less than three conformant devices from different manufacturers found. "
Regards,

**Reply by Je34** [reply]
I found, I have a mistake about the BOSCH camera firmware version. It's OK now, thanks
